windows-nsis/build-snapshot

```diff
@@ -31,7 +32,7 @@ main() {
 	git clone -q --depth 1 --branch "${OPENVPN_BRANCH}" "${OPENVPN_URL}" || die "openvpn.clone"
 	cd openvpn
 	autoreconf -ivf || die "autoreconf.openvpn"
-	./configure --disable-lzo --disable-snappy --disable-lz4 || die "openvpn.configure"
+	./configure --disable-lzo --disable-lz4 || die "openvpn.configure"
```

Comment:
The build-snapshot script is primarily intended for Git "master", although it could be used with release branches. In any case this change is good.

Based on Git logs snappy support was never added to the OpenVPN's "release/2.3" branch (commit 38d96bd7975 in "master"), nor removed from it (commit 9403e3f4b510fb in "master"). I also verified that build-complete does not break with this change. Feel free to create another PR for removing --disable-snappy from openvpn-build's "release/2.3" branch.